modules/ROOT/pages/otel-support.adoc

```diff
@@ -0,0 +1,145 @@
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| Categories | Description | Possible Values
+| Name | Name of the span | “mule:set-variable”
+| Parent Span ID | Span ID of current span's parent | “86838830d494d679”
+| Start and End Timestamps | Earliest and latest timestamp for a span | "2021-10-22 16:04:01.209458162 +0000 UTC"
+| Span Context | Contains data that uniquely identifies the span: trace ID, span ID, trace flags, trace state | 
+| Attributes | Key-value pairs containing metadata used to annotate a span to carry information about the operation it's tracking. | {
+“location”: “flow/processors/2”,
+“correlation.id”:  “1234abcd”
+}
+| Span Events | A structured log message or annotation on a span, used to denote a meaningful, singular point in time during the span's duration. | {
+      "name": "",
+      "message": "OK",
+      "timestamp": "2021-10-22 16:04:01.209512872 +0000 UTC"
+    }
+| Span Links | Links that associate one span with one or more spans, indicating a causal relationship. | 
+| Span Status | Status attached to a span. You can set a span status when there is a known error in the app code, such as an exception. | “ERROR”, “OK”, “UNSET”
+| Span Kind | Provides information about how the trace is assembled. | “CLIENT”, “INTERNAL”, “PRODUCER”
+|===
+
+=== Example of Flow Tracing Feature
+
+When tracing a flow execution, each Mule component execution involved in processing the message is represented as spans. The component spans describe the relevant aspects of the execution, which can range from a single span for simple components, such as the Set Payload, to more complex structures, such as a Batch Job.
+
+.Batch Instance 
+The Mule runtime traces complex components, such as Batch Jobs.
+
+image::otel-batch-instance.png["A batch flow including a Listener and Set Payload components, and a more complex structure"]
+
+.Spans Generated
+Each batch record processed by the first step of the Batch Job is shown in the trace with each record span containing a child span representing the unit of work done by the Logger component.
+Then, the Batch Aggregator span is shown, along with the For Each and the processors executed within it, such as the Transform Message and the Logger. 
+Following this, the trace displays the other two batch steps, along with their respective components.
+Finally, the Batch On Complete span is shown, along with the Logger component that appears after the Batch Job in the flow. 
+
+=== Example of Distributed Tracing Feature
+
+Consider two Mule apps where the first sends a request to the second.
+
+image::otel-mule-app1.png["A flow including a Listener, Logger, and Request components of an HTTP app"]
+
+image::otel-mule-app2.png["A flow showing the process flow for an asynchronous application"]
+
+Even though these are two different flows, distributed tracing captures both as part of the same trace because OpenTelemetry is instrumentation in Mule runtime and HTTP Connector.
```

Comment:
```suggestion
Distributed tracing captures both flows as part of the same trace because OpenTelemetry is instrumented in Mule and HTTP Connector.
```